client/src/partials/cash/cash.js

```diff
@@ -29,15 +29,14 @@ function CashController(Cash, Cashboxes, AppCache, Currencies, Exchange, Session
   if (!cashboxId) {
     $state.go('^.select', {}, { notify : false });
 
-  // if there is no URL id but one in localstorage, the state with the
+  // if there is no URL id but one in localstorage, update the state with the
   // localstorage params
   } else if (cashboxId && !$state.params.id) {
     $state.go('^.window', { id : cashboxId }, { location: 'replace'});
   }
 
   // bind methods
   vm.openInvoicesModal = openInvoicesModal;
-  vm.openTransferModal = openTransferModal;
```

Comment:
It's still a part - I've just moved it to a separate `ui-router` state.  So, instead of `ng-click="ctrl.openTransferModal()"`, we have `<a ui-sref="cash.transfer(ctrl.cashbox.id)">`.  This change was introduced in a previous PR that dealt with the cash client.

For information, see [cash.routes](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/partials/cash/cash.routes.js#L25).